doc/man/sphinx-build.rst

```diff
@@ -64,40 +64,55 @@ Options
 
    .. versionadded:: 1.2.1
 
-.. option:: -b buildername
+.. option:: -b buildername, --builder buildername
 
    Selects a builder.
 
    See :doc:`/usage/builders/index` for a list of all of Sphinx's built-in builders.
    Extensions can add their own builders.
 
-.. option:: -a
+   .. versionchanged:: 7.3
+      Add ``--builder`` long option.
+
+.. option:: -a, --all
 
    If given, always write all output files. The default is to only write output
    files for new and changed source files. (This may not apply to all
    builders.)
 
-.. option:: -E
+   .. versionchanged:: 7.3
+      Add ``--all`` long option.
+
+.. option:: -E, --no-env
```

Comment:
Maybe `--isolate` or `--no-incremental` to mimick Python/mypy options (I think `--no-incremental` is more correct since `--isolate` may assume not looking for the configuration file).

Reply:
Fot Python it's `-I` for "isolated".